src/admin/paths/admin@reports@admin_approve.yaml

```diff
@@ -0,0 +1,55 @@
+post:
```

Comment:
This API canceled right? Or it should be GET

Reply:
Yeah removing it